tern/classes/image.py

```diff
@@ -26,9 +26,9 @@ class Image:
         add_checksums: add new checksums in the existing list of the checksums
         to_dict: return a python dictionary representation of the image
     '''
-    def __init__(self, image_id=None):
+    def __init__(self, repotag=None):
         '''Either initialize using id'''
```

Comment:
Can you change this to `'''Initialize using the image's repo name and tag string'''`?

Reply:
@nishakm shouldn't it be same like `'''Initialize using repotag'''`?